packages/@react-spectrum/tag/test/TagGroup.test.js

```diff
@@ -316,13 +316,24 @@ describe('TagGroup', function () {
     fireEvent.keyUp(document.activeElement, {key: 'Home'});
     expect(document.activeElement).toBe(tags[0]);
 
+    jest.spyOn(HTMLElement.prototype, 'scrollWidth', 'get').mockImplementation(() => 100);
+    jest.spyOn(HTMLElement.prototype, 'offsetWidth', 'get').mockImplementation(function () {
+      return this.getAttribute('role') === 'row' ? 10 : 100;
+    });
+
+    jest.spyOn(HTMLElement.prototype, 'offsetLeft', 'get').mockImplementation(function () {
+      return [...this.parentElement.childNodes].indexOf(this) * 10;
+    });
+
     fireEvent.keyDown(document.activeElement, {key: 'PageDown'});
     fireEvent.keyUp(document.activeElement, {key: 'PageDown'});
-    expect(document.activeElement).toBe(tags[1]);
+    expect(document.activeElement).toBe(tags[3]);
 
     fireEvent.keyDown(document.activeElement, {key: 'PageUp'});
     fireEvent.keyUp(document.activeElement, {key: 'PageUp'});
     expect(document.activeElement).toBe(tags[0]);
+
+    jest.restoreAllMocks();
```

Comment:
not super important, but if this test fails, it will fail possibly every test following it, could wrap this one in a describe block with an afterEach to call the restoreAllMocks?